common/protos-java-kt/README.md

```diff
@@ -5,35 +5,27 @@
 1) Create a Personal Access Token from Github with write/read/delete access to packages.
```

Comment:
Can you add more instructions here of the kind: Sign in to github.com - Select Settings --> Developer Settings --> Personal access tokens --> Generate new token